File: CS/DXSample/ViewModel/SalesViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;
using DXSample.Common;

namespace DXSample.ViewModel {
    public class SalesViewModel {
        public string Content { get; set; }
        public SalesViewModel() {
            Content = "A representation of SalesViewModel.";
            ViewInjectionManager.Default.RegisterNavigatedEventHandler(this, () => {
                ViewInjectionManager.Default.Navigate(Regions.Navigation, ModuleType.Sales);
            });
        }
        public static SalesViewModel Create() {
            return ViewModelSource.Create(() => new SalesViewModel());
        }
    }
}

File: VB/DXSample/ViewModel/SalesViewModel.vb
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.POCO
Imports DXSample.Common

Namespace DXSample.ViewModel

    Public Class SalesViewModel

        Public Property Content As String

        Public Sub New()
            Content = "A representation of SalesViewModel."
            ViewInjectionManager.Default.RegisterNavigatedEventHandler(Me, Sub() ViewInjectionManager.Default.Navigate(Navigation, ModuleType.Sales))
        End Sub

        Public Shared Function Create() As SalesViewModel
            Return ViewModelSource.Create(Function() New SalesViewModel())
        End Function
    End Class
End Namespace
